dist/angular-chart.js

```diff
@@ -311,15 +311,18 @@
       var dataset = {
         labels: labels,
         datasets: [{
-          data: data,
-          backgroundColor: colors.map(function (color) {
-            return color.pointBackgroundColor;
-          }),
-          hoverBackgroundColor: colors.map(function (color) {
-            return color.backgroundColor;
-          })
+          data: data
         }]
       };
+      colors.map(function (color) {
+        Object.keys(color).forEach(function (key) {
```

Comment:
`Object.keys(color)` is going to loop through all the properties of the colors object and create an array from the keys. Doing that on each color iteration is not super efficient, it would be better to have the keys in a constant so that we don't need to repeat that on every iteration (the keys from the color object doesn't don't change).